### ISSUE TITLE
LogServer startmc fails to start server when it fails to start

### ISSUE THREAD

**Issue by NullOrNaN** [CoolV1994/LogViewer#3]
As noted in the title, when issuing a startmc command and it fails to start the log server does not allow you to start the process by thinking that it's already started! Example log below with login information removed

```
[22:29:25 INFO]: [LogViewer] ls_user issued command: logserver startmc
[22:29:25 INFO]: [LogViewer] [Launcher] Starting Minecraft Server... (LogServer)
[22:29:25 INFO]: [LogViewer] ProcessLauncher[commands='[java, -Djline.terminal=jline.UnsupportedTerminal, -Xms8G, -Xmx8G, -jar, Thermos.jar]', directory='/home/flux/servers/flux-ftb']
Error: Unable to access jarfile Thermos.jar
[22:29:25 INFO]: [LogViewer] ServerProcess[commands='[java, -Djline.terminal=jline.UnsupportedTerminal, -Xms8G, -Xmx8G, -jar, Thermos.jar]', isRunning='false']
[22:29:25 WARNING]: [LogViewer] [Launcher] Server ended with bad state (exit code 1)
[22:29:25 INFO]: [LogViewer] ls_user has started the server.
[LogServer] Minecraft server is starting.
[22:34:58 INFO]: [LogViewer] ls_user issued command: logserver startmc
```

**Comment by CoolV1994** [comment]
Thanks for reporting. I will check this out.

**Comment by CoolV1994** [comment]
This has been fixed in version 4.7.2.

**Comment by NullOrNaN** [comment]
Confirmed that this has been fixed on my side as well - thanks!
